tools/fiptool/fiptool.c

```diff
@@ -52,34 +52,6 @@
 #define OPT_PLAT_TOC_FLAGS 1
 #define OPT_ALIGN 2
 
-static image_desc_t *lookup_image_desc_from_uuid(const uuid_t *uuid);
```

Comment:
As @dp-arm states, TF code-base is not consistent in this regard and so this change doesn't bring this file into alignment with the rest of the codebase - if would trade inconsistency from one direction to another!
The rest of the PR looks good - could you update the PR without this change?